spec/index.html

```diff
@@ -282,23 +283,23 @@ <h3>Graph Statements</h3>
 
   </section>
 
-  <section id="quoted-triples">
-    <h3>Quoted Triples</h3>
-
-    <p>TriG shares the same syntax for representing <dfn data-cite="RDF12-CONCEPTS#dfn-quoted-triple">quoted triples</dfn>
-      as Turtle, including the <a data-cite="RDF12-TURTLE#annotation-syntax">Annotation Syntax</a>.</p>
-
-    <pre id="ex-annotated-triple"
-         class="example trig"
-         data-transform="updateExample"
-         title="TriG annotated triples">
-    <!--
-      BASE <http://example.org/>
-      PREFIX : <#>
-      :G {
-        _:a :name "Alice" {| :statedBy :bob |} .
-      }
-    -->
+  <section id="triple-terms">
+    <h3>Triple Terms</h3>
+
+    <p>TriG shares the same syntax for representing <a data-cite="RDF12-CONCEPTS#dfn-triple-term">triple terms</a>
+      as Turtle, including <a data-cite="RDF12-TURTLE#reifying-triples">Reifying Triples</a>
+      and the <a data-cite="RDF12-TURTLE#annotation-syntax">Annotation Syntax</a>.</p>
+
+    <pre id="ex-triple-term" class="example turtle" data-transform="updateExample"
+         title="Triple Term">
+      <!--
+      PREFIX :    <http://www.example.org/>
+      PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+
+      _:e38  :familyName                     "Smith" .
+      _:anno rdf:reifies <<( _:e38 :jobTitle "Designer" )>> .
+      _:anno :accordingTo                     _:e22 .
```

Comment:
When aligning S P O columns, it doesn't make sense to me to align a reified O with unreified Os, so I suggest either —
```suggestion
                             _:e38 :familyName   "Smith" .
      _:anno rdf:reifies <<( _:e38 :jobTitle     "Designer" )>> .
                             _:anno :accordingTo _:e22 .
```